src/views/keyevents/Keybindings.ts

```diff
@@ -124,14 +125,26 @@ export const KeybindingsForMenu: KeybindingMenuType[] = [
     },
     {
         action: KeyboardAction.tabPrevious,
-        accelerator: "CmdOrCtrl+K",
+        accelerator: "Ctrl+Shift+Tab",
     },
     {
         action: KeyboardAction.tabNext,
-        accelerator: "CmdOrCtrl+J",
+        accelerator: "Ctrl+Tab",
     },
     {
         action: KeyboardAction.tabClose,
+        accelerator: "CmdOrCtrl+Shift++W",
```

Comment:
I didn't understood why the Cmd+Tab was hooked to the close pane event in the first place :)

I changed it to the close tab to Cmd+W, and the close pane to Cmd+P - it close to the J/K pane navigation and it usually reserve to send to printer on mac apps, so not much usage on terminal app.